plugin/PDFView.qml

```diff
@@ -190,7 +198,41 @@ SilicaFlickable {
 
     children: [
         HorizontalScrollDecorator { color: Theme.highlightDimmerColor },
-        VerticalScrollDecorator { color: Theme.highlightDimmerColor }
+        VerticalScrollDecorator { color: Theme.highlightDimmerColor },
+        PDFContextMenu {
```

Comment:
Parent PDFView is not necessarily fullscreen. Scroll page down a bit to show toolbar, click on link -> overlay is anchored to top of toolbar. 

Rotating the device while showing overlay flashes the document briefly. Though might be the same issue as application main page not showing while backstepping, likely resulting from scenegraph setup.